src/backend/src/utils/teams.utils.ts

```diff
@@ -72,3 +72,8 @@ export const getTeamsFromUsers = (users: UserWithTeams[]): Team[][] => {
     return teams;
   });
 };
+
+export const removeUsersFromTeam = (currentUsers: User[], usersToRemove: User[]): number[] => {
```

Comment:
this actually removes a list of users from another list of users, so I'd rename it to be more specific to that/ add a comment specifying what it's doing

Reply:
love how clean this makes the code tho